crypto/objects/objects.txt

```diff
@@ -933,11 +933,39 @@ nist_hashalgs 1		: SHA256		: sha256
 nist_hashalgs 2		: SHA384		: sha384
 nist_hashalgs 3		: SHA512		: sha512
 nist_hashalgs 4		: SHA224		: sha224
+nist_hashalgs 5		: id-sha512-224		: sha512-224
+nist_hashalgs 6		: id-sha512-256		: sha512-256
+nist_hashalgs 7		: id-sha3-224		: sha3-224
+nist_hashalgs 8		: id-sha3-256		: sha3-256
+nist_hashalgs 9		: id-sha3-384		: sha3-384
+nist_hashalgs 10	: id-sha3-512		: sha3-512
+nist_hashalgs 11	: id-shake128		: shake128
+nist_hashalgs 12	: id-shake256		: shake256
+nist_hashalgs 13	: id-hmacWithSHA3-224	: hmac-sha3-224
+nist_hashalgs 14	: id-hmacWithSHA3-256	: hmac-sha3-256
+nist_hashalgs 15	: id-hmacWithSHA3-384	: hmac-sha3-384
+nist_hashalgs 16	: id-hmacWithSHA3-512	: hmac-sha3-512
+nist_hashalgs 17	: id-shake128-len	: shake128-len
+nist_hashalgs 18	: id-shake256-len	: shake256-len
```

Comment:
I think the -len values on itself aren't useful. As I understand it, they're incomplete OIDs that need extra parameters.